### ISSUE TITLE
Is the ita_scrapper still working

### ISSUE THREAD

**Issue by haohcraft** [mayanez/flight_scraper#6]
Hi,

I am trying to use the way in your scrapper to fetch the fares.
I tested below at http://www.hurl.it/
by POST

http://matrix.itasoftware.com/xhr/shop/search?name=specificDates&inputs={"slices":[{"origins":["BOS"],"originPreferCity":true,"destinations":["SFO"],"destinationPreferCity":false,"date":"2014-09-10","isArrivalDate":false,"dateModifier":{"minus":0,"plus":0}},{"destinations":["BOS"],"destinationPreferCity":true,"origins":["SFO"],"originPreferCity":false,"date":"2014-09-18","isArrivalDate":false,"dateModifier":{"minus":0,"plus":0}}],"pax":{"adults":1},"cabin":"COACH","changeOfAirport":true,"checkAvailability":true,"page":{"size":30},"sorts":"default"}&summarizers=carrierStopMatrix,currencyNotice,solutionList,itineraryPriceSlider,itineraryCarrierList,itineraryDepartureTimeRanges,itineraryArrivalTimeRanges,durationSliderItinerary,itineraryOrigins,itineraryDestinations,itineraryStopCountList,warningsItinerary&format=JSON

 with header

Host                             matrix.itasoftware.com
Content-Type              application/x-www-form-urlencoded
Cache-Control             no-cache
Content-Length           0

But I got a response like:

Alternate-Protocol: 80:quic
Content-Length: 0
Content-Type: text/plain; charset=UTF-8
Date: Tue, 19 Aug 2014 14:29:33 GMT
Location: http://matrix.itasoftware.com/xhr/shop/search?name=specificDates&summarizers=carrierStopMatrix%2CcurrencyNotice%2CsolutionList%2CitineraryPriceSlider%2CitineraryCarrierList%2CitineraryDepartureTimeRanges%2CitineraryArrivalTimeRanges%2CdurationSliderItinerary%2CitineraryOrigins%2CitineraryDestinations%2CitineraryStopCountList%2CwarningsItinerary&format=JSON&inputs=%7B%22slices%22%3A%5B%7B%22origins%22%3A%5B%22BOS%22%5D%2C%22originPreferCity%22%3Atrue%2C%22destinations%22%3A%5B%22SFO%22%5D%2C%22destinationPreferCity%22%3Afalse%2C%22date%22%3A%222014-09-10%22%2C%22isArrivalDate%22%3Afalse%2C%22dateModifier%22%3A%7B%22minus%22%3A0%2C%22plus%22%3A0%7D%7D%2C%7B%22destinations%22%3A%5B%22BOS%22%5D%2C%22destinationPreferCity%22%3Atrue%2C%22origins%22%3A%5B%22SFO%22%5D%2C%22originPreferCity%22%3Afalse%2C%22date%22%3A%222014-09-18%22%2C%22isArrivalDate%22%3Afalse%2C%22dateModifier%22%3A%7B%22minus%22%3A0%2C%22plus%22%3A0%7D%7D%5D%2C%22pax%22%3A%7B%22adults%22%3A1%7D%2C%22cabin%22%3A%22COACH%22%2C%22changeOfAirport%22%3Atrue%2C%22checkAvailability%22%3Atrue%2C%22page%22%3A%7B%22size%22%3A30%7D%2C%22sorts%22%3A%22default%22%7D&cc
Server: Apache
Set-Cookie: PREF="ID=893bb06e583439e19fc9ab4a5096d4e5ba047dfa7df8c2c5fd841d7abe95ac56e54bd54689c1e3b0aff4edbddcfb3a5ca581d7ade8ba772efdeb8d833d10b453:TM=1408458573:S=6HGECk7Jzb068IDO"; Version=1

Do you know there is something wrong with my POST request??

Thanks a lot.

**Comment by mayanez** [comment]
It should still be working assuming itasoftware hasn't changed, which from my preliminary tests doesn't seem like it. I can look into it more later this week as I've been quite busy recently hence why I kinda put further development on hold. I'll keep you posted on what I find.

**Comment by mayanez** [comment]
I got it to work fine. Seems like the have a few 302's setup on their domain but if you put hurl.it to follow 302 should work fine. Just for reference here are the settings I used:

Follow redirects: On
POST

Headers---
Host: matrix.itasoftware.com
Content-Type:application/x-www-form-urlencoded
Cache-Control: no-cache
Content-Length: 0

Parameters---
name: specificDatesSlice
summarizers: sliceSelections,carrierStopMatrixSlice,currencyNotice,solutionListSlice,priceSliderSlice,carrierListSlice,departureTimeRangesSlice,arrivalTimeRangesSlice,durationSliderSlice,originsSlice,destinationsSlice,stopCountListSlice,warningsSlice
format: JSON
inputs: {"slices":[{"origins":["SEA"],"originPreferCity":true,"destinations":["NYC"],"destinationPreferCity":true,"date":"2014-09-19","isArrivalDate":false,"dateModifier":{"minus":0,"plus":0},"timeRanges":[{"min":"17:00","max":"21:00"},{"min":"21:00","max":"23:59"}]},{"destinations":["SEA"],"destinationPreferCity":true,"origins":["NYC"],"originPreferCity":true,"date":"2014-09-26","isArrivalDate":false,"dateModifier":{"minus":0,"plus":0},"timeRanges":[{"min":"17:00","max":"21:00"},{"min":"21:00","max":"23:59"}]}],"pax":{"adults":1},"cabin":"COACH","changeOfAirport":true,"checkAvailability":true,"sliceIndex":0,"page":{"size":30},"sorts":"default"}

I'll go ahead and close this since there doesn't seem to be an issue. Feel free to open another or email me with any questions if you have problems.
